pulp_ansible/app/models.py

```diff
@@ -45,44 +43,72 @@ class Meta:
         unique_together = ("namespace", "name")
 
 
+class Tag(Model):
+    """A model representing a Tag.
+
+    Fields:
+
+        name (models.CharField): The Tag's name.
+    """
+
+    name = models.CharField(max_length=32, unique=True, editable=False)
+
+
 class CollectionVersion(Content):
     """
     A content type representing a Collection.
 
+    This model is primarily designed to adhere to the data format for Collection content. That spec
+    is here: https://docs.ansible.com/ansible/devel/dev_guide/collections_galaxy_meta.html
+
     Fields:
 
-        version (models.CharField): Version string.
-        metadata: Artifact metadata in JSON format.
-        contents: Contnet items (e.g. roles, modules) provided by
-            collection in JSON format.
-        quality_score: Total version quality score,
-            floating-point number in range [0, 5].
+        authors (psql_fields.ArrayField): A list of the CollectionVersion content's authors.
+        contents (psql_fields.JSONField): A JSON field with data about the contents.
+        dependencies (psql_fields.JSONField): A dict declaring Collections that this collection
+            requires to be installed for it to be usable.
+        description (models.TextField): A short summary description of the collection.
+        docs_blob (psql_fields.JSONField): A JSON field holding the various documentation blobs in
+            the collection.
+        documentation (models.URLField): The URL to any online docs.
+        homepage (models.URLField): The URL to the homepage of the collection/project.
+        issues (models.URLField): The URL to the collection issue tracker.
+        license (psql_fields.ArrayField): A list of licenses for content inside of a collection.
+        license_file (models.CharField): The path to the license file for the collection.
+        name (models.CharField): The name of the collection.
+        namespace (models.CharField): The namespace of the collection.
+        repository (models.URLField): The URL of the originating SCM repository.
+        version (models.CharField): The version of the collection.
 
     Relations:
 
-        collection: Reference to a collection model.
+        collection (models.ForeignKey): Reference to a collection model.
+        dependencies (models.ManyToManyField): An asymmetric reference to other CollectionVersion
```

Comment:
@bmbouter I assume this should be removed from the relations docstring

Reply:
yup this is wrong. Thank you! Will fix now and push.